### ISSUE TITLE
Add eslint-plugin-import.

### ISSUE THREAD

**Issue by parkerziegler** [FormidableLabs/renature#108]
This is a simple one — rather than manually maintaining import rules, we should just add [`eslint-plugin-import`](https://github.com/benmosher/eslint-plugin-import) to handle this for us.

The most important rules for us are: 

- [`order`](https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/order.md)
- [`newline-after-import`](https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/newline-after-import.md)

**Comment by jzandag** [comment]
can I try doing this?

**Comment by parkerziegler** [comment]
Hey @jzandag, sure thing. Here are the steps I think we need to take:

1. Add `eslint-plugin-import` to `devDependencies`.
2. Add the default `extends` from that plugin, in addition to its TypeScript support. This will amount to adding these three lines in the `extends` array in our root `.eslintrc.js`:

```js
'plugin:import/errors',
'plugin:import/warnings',
'plugin:import/typescript',
```

These should go after `'plugin:react/recommended'` and before `'prettier'`.

3. Add the following lines to the `rules` array:

```js
'import/newline-after-import': 2,
'import/order': [
  'error',
  {
    'newlines-between': 'always',
  },
],
```

4. Run `yarn lint --fix` to fix any import errors we may get and commit the results along with the above changes!

Does that work for you?

**Comment by jzandag** [comment]
Thanks for the detailed instructions! I'll try doing it now

**Comment by parkerziegler** [comment]
Fixed by #110
